Index: libkgeomap/tests/CMakeLists.txt
===================================================================
--- libkgeomap.orig/tests/CMakeLists.txt	2015-04-23 21:05:46.627670385 +0200
+++ libkgeomap/tests/CMakeLists.txt	2015-04-24 13:43:49.021982954 +0200
@@ -10,6 +10,7 @@
 
     include(CTest)
     enable_testing()
+    add_definitions("-DTEST_DATA='\"${CMAKE_CURRENT_SOURCE_DIR}/data/\"'")
 
     # test the primitive datatypes and helper functions
     set(test_primitives_sources
Index: libkgeomap/tests/test_tracks.cpp
===================================================================
--- libkgeomap.orig/tests/test_tracks.cpp	2015-04-23 23:57:43.006997678 +0200
+++ libkgeomap/tests/test_tracks.cpp	2015-04-24 13:44:44.079776907 +0200
@@ -47,10 +47,8 @@
  */
 KUrl GetTestDataDirectory()
 {
-    // any better ideas on how to get the path?
-    const KUrl thisCPPFile(__FILE__);
-    KUrl testDataDir = thisCPPFile.upUrl();
-    testDataDir.addPath("data/");
+    KUrl testDataDir(TEST_DATA);
+
     return testDataDir;
 }
 
